app/Components/Helpers/HomeHelper.js

```diff
@@ -0,0 +1,154 @@
+ export const checkDatabaseForSearch = (search, setState, scrubbedTweets) => {
```

Comment:
The name 'helpers' becomes a 'junk drawer' - I would personally consider splitting the fetch calls and the cleaners apart and name the files a little more specifically - but that's just a suggestion